README.md

```
@@ -279,24 +368,150 @@ For `--lightroom` and `--lightroomimport`, the script does more work to identify
+
+### Checking what you have
+
+```bash
+exiftool -ver          # your ExifTool, e.g. 13.59
+./stackcopy.py --version   # Stackcopy 1.6.0
+```
+
+Every Lightroom-mode run also says so itself, on stderr, before it starts
+scanning:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)